app/src/views/DrefApplicationForm/Operation/index.tsx

```diff
@@ -181,6 +182,26 @@ function Operation(props: Props) {
         setSelectedIntervention(undefined);
     }, [setFieldValue, setSelectedIntervention]);
 
+    const onPopulationChange = useCallback((
+        val: number | undefined,
+        name: 'men' | 'women' | 'girls' | 'boys',
+    ) => {
+        setValue(
+            (oldValue: PartialDref | undefined) => {
+                const newValue = {
+                    ...oldValue,
+                    [name]: val,
+                };
+                return {
+                    ...newValue,
+                    ...calculateTotalTargetedPopulation(newValue),
```

Comment:
Use the value directly instead of spreading an object. The function should return the total value.